android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMHelpFileActivity.java

```diff
@@ -132,7 +147,7 @@ public void onPageFinished(WebView view, String url) {
         // Inject a meta viewport tag into the head of the file if it doesn't exist
         webView.loadUrl(
           "javascript:(function() {" +
-            "if(!document.querySelectorAll('meta[name=viewport]').length) {"+
+            "if(document.head && !document.querySelectorAll('meta[name=viewport]').length) {"+
```

Comment:
Hmm... this is basically a duplicate of https://github.com/keymanapp/keyman/pull/4531/files#r584485255's block, right?

I feel that lends more weight to the idea of abstracting it into a common function.

Reply:
Yeah, I'll refactor them to a helper method